.github/workflows/.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
```

Comment:
- [ ] Here also, The file should be in the root of your repository. Kindly add it there as its placed on the linter file requirements 👍